apps/docs/src/app/platform/component-docs/platform-multi-combobox/platform-multi-combobox-docs.component.html

```diff
@@ -0,0 +1,71 @@
+<fd-docs-section-title id="data-source" componentName="multi-combobox">DataSource</fd-docs-section-title>
+<description>
+    Input <code>[dataSource]</code> has this type <code>type FdpListDataSource&lt;T[]&gt; = ListDataSource&lt;T[]&gt; | T[]</code>
+</description>
+<component-example>
+    <fdp-multi-combobox-datasource-example></fdp-multi-combobox-datasource-example>
+</component-example>
+<code-example [exampleFiles]="multiComboboxDatasourceExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="mobile" componentName="multi-combobox">Multi-Combobox Mobile Mode</fd-docs-section-title>
+<description>
+    To use Multi-Combobox in mobile mode:
+    <ul>
+        <li>import <code>MultiComboboxMobileModule</code></li>
+        <li>Set <code>[mobile]</code> property to true</li>
+        <li>Set <code>[mobileConfig]</code> with <code>MobileModeConfig</code> configuration object to customize the
+            look of the control
+        </li>
+    </ul>
+
+    Multi-Combobox Mobile Mode can be configured <strong>globally</strong> by providing <code>MOBILE_MODE_CONFIG</code> token.
+    Check <a [routerLink]="['/core/global-config']" fragment="mobile-mode-config">Global Configuration page</a> for more
+    information.
+</description>
+<component-example>
+    <fdp-multi-combobox-mobile-example></fdp-multi-combobox-mobile-example>
+</component-example>
+<code-example [exampleFiles]="multiComboboxMobileExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="group" componentName="multi-combobox">
+    Multi-Combobox with Groups
+</fd-docs-section-title>
+<description>
+    Multi-Combobox with Groups.
+    Note that a <code>group</code> and <code>groupKey</code> is required
+</description>
+<component-example>
+    <fdp-multi-combobox-group-example></fdp-multi-combobox-group-example>
+</component-example>
+<code-example [exampleFiles]="multiComboboxGroupExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="columns" componentName="multi-combobox">
+    Multi-Combobox with Two Columns
+</fd-docs-section-title>
+<description>
+    Multi-Combobox with two columns of information.
+    Note that a <code>secondaryKey</code> and <code>shoSecondaryText</code> is required
+</description>
+<component-example>
+    <fdp-multi-combobox-columns-example></fdp-multi-combobox-columns-example>
+</component-example>
+<code-example [exampleFiles]="multiComboboxColumnsExample"></code-example>
```

Comment:
Groups example here, not columns example.

<img width="808" alt="image" src="https://user-images.githubusercontent.com/20265336/115013202-6bd21300-9eb9-11eb-97fe-d587a7beb4f5.png">